### DISCUSSION TITLE
Best way to scale scraping

### DISCUSSION THREAD

**Discussion by Cvoluj** [kaliiiiiiiiii/Selenium-Driverless#282] [Q&A] [answered]
I saw there are some ways to scale: `new_context` and `multithreading`. So I'm curious what will be the best.

**Comment by ttlns** [comment, marked as answer]
Uh combination of both, I suppose.
With new_context, python can reach its limits at some point
=> new_context + multiprocessing

**Reply by Cvoluj** [reply]
Thanks a lot, for answer and great library
